examples/get-started/first_figure.py

```diff
@@ -0,0 +1,159 @@
+# and ``water`` to set the colors on the figure.
+#
+# When plotting colors in PyGMT, there are multiple
+# :gmt-docs:`color codes <gmtcolors.html>`, that can be used. This includes
+# standard GMT color names (like ``skyblue``), R/G/B levels (like ``0/0/255``),
+# a hex value (like ``#333333``), and a graylevel (like ``50``). For this
+# example, GMT color names are used.
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+)
+fig.show()
+
+###############################################################################
+# Set the projection
+# ------------------
+#
+# This figure now has its colors set, but there is not projection or size
+# set for the map. Both of these values are set using the ``projection``
+# parameter.
+#
+# The appropriate projection varies for the type of map. The available
+# projections are explained in the :doc:`projection </projections/index>`
+# gallery. For this example, the Mercator projection is set using ``"M"``.
+# The width of the figure will be 10 centimeters, as set by ``"10c"``. The map
+# size can also be set in inches using "i" (e.g. a 5 inch wide Mercator
+# projection would use ``"M5i"``).
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+    projection="M10c",
+)
+fig.show()
+
+###############################################################################
+# Add a frame
+# -----------
+#
+# While that the map's colors, projection, and size have been set, the region
+# that is being displayed is not apparent. A frame can be added to
+# annotate the latitude and longitude of the region.
+#
+# The ``frame`` parameter is used to add a frame to the figure. For now, it
+# will be set to ``a`` to use default settings, but later tutorials will
+# show how ``frame`` can be used to customize the axes, gridlines, and titles.
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+    projection="M10c",
+    frame="a",
+)
+fig.show()
+
+###############################################################################
+# Additional exercises
+# --------------------
+#
+# This is the end of the first tutorial. Here are some additional exercises
+# for the concepts that were discussed:
+#
+# 1. Make a map of Germany using its DCW ISO code ("DE"). Pass the ISO code
+#    as a Python string to the ``"region"`` parameter.
+#
+# 2. Change the color of the land to "khaki" and the water to "azure".
+#
+# 3. Change the color of the lakes (using the ``"lakes"`` parameter) to "red."
+#
+# 4. Create a global map. Set the region to "d" to center the map at the Prime
+#    Meridian or "g" to center the map at the International Date Line. When the
+#    region is set without using a list full of integers or floating numbers,
```

Comment:
> When the region is set without using a list full of integers or floating numbers, the argument needs to be passed as a Python string.

If I understand you correctly, you mean passing a string like `-20/20/-30/30` to `region`, right? I don't think it's clearly explained by this sentence because it doesn't say that the string should contain slash-separated numbers.

Reply:
I'm actually referring to passing `d` or `g` when it comes to region. While I know that the literal GMT string can be passed for region, I don't want to make a mention of it in the first section of the tutorial. I think bringing up too many options early on is confusing, and using a list for something like this is probably more familiar to Python users instead of a string with numbers.

Reply:
I see what you meant. I also agree that a string like `-20/20/-30/30` is not Pythonic and should not be covered in this starter tutoria.

Reply:
Actually, you could pass in a list of strings like `["10W", "20E", "30S", "40N"]` right? Not necessarily all numbers.

Reply:
> Actually, you could pass in a list of strings like `["10W", "20E", "30S", "40N"]` right? Not necessarily all numbers.

Yes, but it's not commonly used, and better not to introduce too many options.